app/workers/transactional_email_analytics_job.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
```

Comment:
This is a renaming of `app/workers/hca/submission_failure_email_analytics_job` and not an entirely new class

Reply:
sidekiq jobs should be renamed with both classes present, if a `HCA::SubmissionFailureEmailAnalyticsJob` is queued then it will try to run that class and won't know that it's been renamed

Reply:
Good catch.

Reply:
I restored the previous version (and its spec).